### DISCUSSION TITLE
can't use google DNS

### DISCUSSION THREAD

**Discussion by rayqusa** [crazy-max/docker-cloudflared#102] [Q&A]
1. insert `https://dns64.dns.google/dns-query{?dns},https://dns.google/dns-query`
2. on pihole going showing no data

but don't get data or CNAME from google DNS

![Screenshot 2023-02-19 at 06 51 11](https://user-images.githubusercontent.com/9047814/219904804-1d65d779-ede3-4487-9f6d-9d3254cafeba.png)

**Comment by crazy-max** [comment]
Don't think this is related to this image but cloudflared itself. If you think this is a bug please create a proper [bug report](https://github.com/crazy-max/docker-cloudflared/issues/new?assignees=&labels=kind%2Fbug%2Cstatus%2Ftriage&projects=&template=bug.yml) with logs and compose stack. Thanks.
